javascripts/2048.js

```diff
@@ -69,76 +72,279 @@ function emptyCell(openCells, usedCells, cellSpace) {
   openCells.push(cellSpace);
 }
 
-// function moveTile(tile, direction) {
-//   //only be user * 2 if the tiles merged
-//   var new_tile_value = tile.attr("data-val") * 2;
-//   tile.attr("data-val", new_tile_value);
-//   tile.text(new_tile_value);
-//
-//   switch(direction) {
-//     case 38: //up
-//       tile.attr("data-row","r0");
-//       break;
-//     case 40: //down
-//       tile.attr("data-row","r3");
-//       break;
-//     case 37: //left
-//       tile.attr("data-col","c0");
-//       break;
-//     case 39: //right
-//       tile.attr("data-col","c3");
-//       break;
-//   }
-// }
+
+// MERGING TILES --------------------------------------------------------------------------
+function merge(tile1, tile2, direction, openCells, usedCells) {
+  if (direction == "up" || direction == 38) {
```

Comment:
Good thought to convert a key code into something human readable. A simpler way to do this would be to define a global const object like so:
```
const DIRECTION = {
  up: 38,
  down: 40,
  left: 37,
  right: 39
};
```

And then you'd be able to check direction by doing this:
`if (direction == DIRECTION.up)`